File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/System/DrugKinkiMessageInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.System
{
    [Serializable]
    public class DrugKinkiMessageInfo
    {
        private String _drugId;
        private String _branchNumber;
        private String _warningLevel;
        private String _kinkiId;
        private String _message;
        private String _category;

        public String DrugId
        {
            get { return this._drugId; }
            set { this._drugId = value; }
        }

        public String BranchNumber
        {
            get { return this._branchNumber; }
            set { this._branchNumber = value; }
        }

        public String WarningLevel
        {
            get { return this._warningLevel; }
            set { this._warningLevel = value; }
        }

        public String KinkiId
        {
            get { return this._kinkiId; }
            set { this._kinkiId = value; }
        }

        public String Message
        {
            get { return this._message; }
            set { this._message = value; }
        }

        public String Category
        {
            get { return this._category; }
            set { this._category = value; }
        }

        public DrugKinkiMessageInfo() { }

        public DrugKinkiMessageInfo(String drugId, String branchNumber, String warningLevel, String kinkiId, String message, String category)
        {
            this._drugId = drugId;
            this._branchNumber = branchNumber;
            this._warningLevel = warningLevel;
            this._kinkiId = kinkiId;
            this._message = message;
            this._category = category;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/system/DrugKinkiMessageInfo.java
package nta.med.data.model.ihis.system;

import java.math.BigDecimal;

public class DrugKinkiMessageInfo implements DrugKinkiInterface   {
	private String drugId ;
	
	private String branchNumber ;
	private BigDecimal warningLevel ;
	private String kinkiId ;
	private String message ;
	private String category ;
	public DrugKinkiMessageInfo(String drugId, String branchNumber,
			BigDecimal warningLevel, String kinkiId, String message, String category) {
		super();
		this.drugId = drugId;
		this.branchNumber = branchNumber;
		this.warningLevel = warningLevel;
		this.kinkiId = kinkiId;
		this.message = message;
		this.category = category;
	}
	public String getDrugId() {
		return drugId;
	}
	public void setDrugId(String drugId) {
		this.drugId = drugId;
	}
	public String getBranchNumber() {
		return branchNumber;
	}
	public void setBranchNumber(String branchNumber) {
		this.branchNumber = branchNumber;
	}
	public BigDecimal getWarningLevel() {
		return warningLevel;
	}
	public void setWarningLevel(BigDecimal warningLevel) {
		this.warningLevel = warningLevel;
	}
	public String getKinkiId() {
		return kinkiId;
	}
	public void setKinkiId(String kinkiId) {
		this.kinkiId = kinkiId;
	}
	public String getMessage() {
		return message;
	}
	public void setMessage(String message) {
		this.message = message;
	}
	public String getCategory() {
		return category;
	}
	public void setCategory(String category) {
		this.category = category;
	}
	public String convertToRowCsv() {
		StringBuilder rowCsv = new StringBuilder();
		rowCsv.append(drugId );
		rowCsv.append(",");
		rowCsv.append(branchNumber);
		rowCsv.append(",");
		rowCsv.append(warningLevel);
		rowCsv.append(",");
		rowCsv.append(kinkiId);
		rowCsv.append(",");
		rowCsv.append(message);
		rowCsv.append(",");
		rowCsv.append(category);

		return rowCsv.toString();
	}

	@Override
	public String[] convertItemToArray() {
		return (new String[]{drugId, branchNumber, String.valueOf(warningLevel), kinkiId, message, category});
	}
}
